omod/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ConceptController1_8Test.java

```diff
@@ -414,7 +408,7 @@ public String getUuid() {
 	 */
 	@Override
 	public long getAllCount() {
-		return 34;
```

Comment:
Shouldn't the test expect a specific count instead of depending on whatever getAllConcepts will return?